Core.ApplicationServices/RightsHolders/ItInterfaceRightsHolderService.cs

```diff
@@ -138,5 +138,48 @@ public class ItInterfaceRightsHolderService : BaseRightsHolderService, IItInterf
                     }
                 );
         }
+
+        public Result<ItInterface, OperationError> UpdateItInterface(Guid interfaceUuid, RightsHolderItInterfaceUpdateParameters updateParameters)
+        {
+            if (updateParameters == null)
+                throw new ArgumentNullException(nameof(updateParameters));
+
+            using var transaction = _transactionManager.Begin(IsolationLevel.ReadCommitted);
+            try
+            {
+                var exposingSystem = _systemService.GetSystem(updateParameters.ExposingSystemUuid);
+
+                if (exposingSystem.Failed)
+                {
+                    if (exposingSystem.Error.FailureType == OperationFailure.NotFound) //If we can't find the exposing system the call will never work and should return BadInput.
+                        return new OperationError("Invalid exposing system id provided", OperationFailure.BadInput);
+                    return exposingSystem.Error;
+                }
+
+                var result = _itInterfaceService
+                    .GetInterface(interfaceUuid)
+                    .Bind(itInterface => _itInterfaceService.UpdateNameAndInterfaceId(itInterface.Id, updateParameters.Name, updateParameters.InterfaceId))
+                    .Bind(ItInterface => _itInterfaceService.UpdateExposingSystem(ItInterface.Id, exposingSystem.Value.Id))
+                    .Bind(itInterface => _itInterfaceService.UpdateVersion(itInterface.Id, updateParameters.Version))
```

Comment:
Husk at trække alle de updates ud der defineres fælles for din fællesmængde-interface (se i rightsholderitsystemservice hvad jeg mener)